### PR TITLE
Errors

### PR DESCRIPTION
**Pull request by Pomax**
addresses #1922

### CODE REVIEW

**Comment by thisandagain** [comment]
In most instances I'm having trouble triggering an error state. @Pomax how were you testing this?

File: www_src/pages/make/make.jsx [hunk 1]
```diff
@@ -24,15 +26,14 @@ var Make = React.createClass({
     this.load();
   },
   onError: function (err) {
-    console.error(err);
+    reportError("Error loading project", err);
```
Comment (thisandagain) [inline www_src/pages/make/make.jsx]: This should be "Error loading projects" (plural)

File: www_src/pages/make/make.jsx [hunk 2]
```diff
@@ -24,15 +26,14 @@ var Make = React.createClass({
...
     this.setState({loading: false});
   },
   onEmpty: function () {
-    console.log('No projects found');
+    reportError('No projects found');
```
Comment (thisandagain) [inline www_src/pages/make/make.jsx]: We should not show an error when no projects exist as it makes for a poor first-time-use.

**Comment by thisandagain** [comment]
This is great. I love how clean the `snackbar` component turned out. Two really minor comments, otherwise R+

File: www_src/pages/make/make.jsx (hunk 2 above)
Comment (Pomax) [inline www_src/pages/make/make.jsx]: :+1:

**Comment by thisandagain** [comment]
RRRRRRR+

**Pull request merged**